src/features/creators/templates/newPackageTemplate/README.md

```diff
@@ -0,0 +1,44 @@
+# <project_name>
+
+> TODO: Write a short description of your project and what it does.
+
+## Table of Contents
+
+- [\<project\_name\>](#project_name)
```

Comment:
need to fix this as it won't get replaced correctly on the find and replace call